src/healthcheck_httpd.erl

```diff
@@ -77,22 +77,43 @@ init(Req, Opts) ->
 %%% private
 %%%===================================================================
 
-read_calver(Path) ->
+read_semver_file(Path, Number) ->
```

Comment:
Pls refactor file reading operations. Currently this implementation re-reads version file 3 times for each /healthcheck call. This should be a single call for SEMVER file per requrest.

Reply:
fixed